testing_programs.csv

```diff
@@ -0,0 +1,12900 @@
+key,tag,outcome,other,topic

```

Comment:
We definitely don't want this in version control.

Reply:
Ok 👍 

I'll take a look at how to excise it from Git while still letting it work in Travis CI

Reply:
thats not what i'm saying. I mentioned I was going to work on this. Also, I was saying we should limit this file, as we don't need this large of a test file. Also, it has "real" data in it that we don't want shown to the world